examples/plot_classification_methods.py

```diff
@@ -138,6 +138,19 @@
 # achieve more than an 80% of score, but the most robust ones are KNN and
 # Parameterized Functional QDA.
 
+##############################################################################
```

Comment:
Can't this be generated programatically? I don't want it to be desynchronized with the code.